SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -8,12 +8,34 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
     private val dispatchers = AppDispatchersImpl()
     private var view: ApplicationContract.View? = null
     private val job: Job = SupervisorJob()
+    private val stations = listOf(
+        "BON",
+        "KGX",
+        "EUS",
+        "MAN",
+        "EDB"
+    )
+
+    private val stationSubmitButtonText = "Search"
 
     override val coroutineContext: CoroutineContext
         get() = dispatchers.main + job
 
     override fun onViewTaken(view: ApplicationContract.View) {
         this.view = view
-        view.setLabel(createApplicationScreenMessage())
+        view.populateOriginAndDestinationSpinners(stations)
+        view.setStationSubmitButtonText(stationSubmitButtonText)
+        view.setStationSubmitButtonHandler()
+    }
+
+    override fun onStationSubmitButtonPressed() {
+        // get inputs
+        var originStation = view?.getOriginStation()
+        var destinationStation = view?.getDestinationStation()
+        // talk to api
+        // send output
+        view?.openUrl("https://www.lner.co.uk/travel-information/travelling-now/live-train-times/depart/${originStation}/${destinationStation}/#LiveDepResults")
```

Comment:
Is this a placeholder before we add talking to the API?

Reply:
yes. the comments are now removed (on the other branch) and replaced with the actual functionality.